CHANGELOG.md

```diff
@@ -25,6 +25,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] - 2022-08-08
+
+### Changed
+- Changed the wrapper scripts to depend on a file named ``/opt/cray/etc/sat/version``
+  to supply the version of the SAT container image, so that there is no longer
+  a hard-coded default SAT version in the script.
```

Comment:
I would suggest editing to describe the default behavior to use "csm-latest" tag.

Reply:
Fixed in 82e4438e7ef1dd0bc7e6c7efb530a7a7167f65ef.